guides/source/api_app.md

```diff
@@ -97,11 +96,10 @@ Handled at the Action Pack layer:
   means not having to spend time thinking about how to model your API in terms
   of HTTP.
 - URL Generation: The flip side of routing is URL generation. A good API based
-  on HTTP includes URLs (see [the GitHub Gist API](http://developer.github.com/v3/gists/)
+  on HTTP, includes URLs (see [the GitHub Gist API](http://developer.github.com/v3/gists/)
```

Comment:
Actually, HTTP vs HTTP includes URLs is ambiguous here. I am sure we are not referring to HTTP includes here.